lambdatimes/src/components/Header.js

```diff
@@ -1,15 +1,46 @@
 import React from 'react';
+import styled, {css} from 'styled-components';
```

Comment:
You aren't utilizing the named {css} import from styled components in this file so you can omit it from the import and use import styled from "styled- components"